src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -97,56 +81,69 @@ const Contributors = ({
 
       <Content>
         <Breadcrumbs slug={location.pathname} />
-        <h1>
+        <ContentHeading as="h1">
           <Translation id="page-contributing-translation-program-contributors-title" />
-        </h1>
-        <h4>
-          <strong>
+        </ContentHeading>
+        <ContentHeading as="h4">
+          <Text as="strong">
             <Translation id="page-contributing-translation-program-contributors-number-of-contributors" />{" "}
             {translatorData.length}
-          </strong>
-        </h4>
-        <p>
+          </Text>
+        </ContentHeading>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-1" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-2" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-contributing-translation-program-contributors-our-translators-3" />
-        </p>
-        <p>
+        </Text>
+        <Text>
           <Translation id="page-languages-interested" />{" "}
           <Link to="/contributing/translation-program/">
             <Translation id="page-languages-learn-more" />
           </Link>
           .
-        </p>
-        <h2>
+        </Text>
+        <ContentHeading as="h2">
```

Comment:
To go with the comment on the [ContentHeading component](https://github.com/ethereum/ethereum-org-website/pull/9369/files#diff-fea7d7fee9efeaf79d0def83b648b3851e5ae6ef52069d898bcb37b54fb24341R27)

```suggestion
        <ContentHeading fontSize={{ base: "2xl", md: "2rem" }} fontWeight={600}>
```